src/Jab/Attributes.cs

```diff
@@ -94,7 +94,7 @@ public ScopedAttribute(Type serviceType, Type implementationType)
         }
     }
 
-#if GENERIC_ATTRIBUTES
```

Comment:
Shouldn't https://github.com/pakrym/jab/blob/621a544e785ba6a0664a09155c0f84d87d117d34/src/Jab/build/Jab.targets#L3C1-L3C1 handle this?

Reply:
yes, I think so